.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    # branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: "test upload via action"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install python-build and twine
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install build twine
+          python -m pip list
+
+      - name: Build a wheel and a sdist
+        run: |
+          PYTHONWARNINGS=error,default::DeprecationWarning python -m build tests/debug_project
+          mv tests/debug_project/dist .
+
+      - name: Verify the distribution
+        run: twine check --strict dist/*
+
+      - name: List contents of sdist
+        run: python -m tarfile --list dist/debug-*.tar.gz
+
+      - name: List contents of wheel
+        run: python -m zipfile --list dist/debug-*.whl
+
+      - name: Test upload
+        uses: ./
+        with:
+          artifacts_path: dist
+          anaconda_nightly_upload_token: ${{ secrets.UPLOAD_TOKEN }}
```

Comment:
@stefanv @jarrodmillman As only org owners have access to the GitHub Actions secrets, can one of you please go to https://github.com/scientific-python/upload-nightly-action/settings/secrets/actions and add a new secret named `UPLOAD_TOKEN` that has the value of the Anaconda Clout API token named `github-actions` on https://anaconda.org/scientific-python-nightly-wheels/settings/access ?

Reply:
Gentle ping on this, as it is needed for testing.

Reply:
Sorry, looking into it now.

Reply:
Nothing to be sorry about!

Reply:
Done.

Reply:
@jarrodmillman Hm. We can follow up on this tomorrow or next week, but for some reason the `UPLOAD_TOKEN` secret you set isn't getting picked up as

https://github.com/scientific-python/upload-nightly-action/blob/a2ba178f650e68bd6f9d8e2b7d59d16ffd257c29/cmd.sh#L18-L25

is erroring out in the CI with

```
...
+ echo 'Getting anaconda token from github secrets...'
+ ANACONDA_ORG=scientific-python-nightly-wheels
Getting anaconda token from github secrets...
+ ANACONDA_TOKEN=
+ '[' -z '' ']'
+ echo 'ANACONDA_TOKEN is empty , exiting...'
ANACONDA_TOKEN is empty , exiting...
+ exit -1
```

Reply:
![2023-07-20T21:44:12,772309624-07:00](https://github.com/scientific-python/upload-nightly-action/assets/123428/8781e4db-0043-4ab2-b6a3-9cbb11dccc38)

The `ANACONDA_TOKEN` environmental secret already existed.

I added the secret as `UPLOAD_TOKEN`  repository secret.

Reply:
> I added the secret as `UPLOAD_TOKEN` repository secret.

Yup, that was the right thing to do. Hm, strange. I'll think on this tomorrow.

Reply:
I just readded the secret to make sure I put in the correct one.

Reply:
@jarrodmillman ah no sorry I wasn't thinking. You did everything right, but this PR is coming from my **fork** and so as there isn't already a workflow to run locally this is using my fork's secret credentials which are missing this information. I just verified this by pushing a `debug/is-this-repo-view-related` branch to this repo and it sees the secret correctly (:+1:).

However, this is still failing given https://github.com/Anaconda-Platform/anaconda-client/issues/676 with

```
...
Uploading wheels to anaconda.org...
Using Anaconda API: https://api.anaconda.org/
Using "scientific-python-nightly-wheels" as upload username
Processing "dist/test_package-0.0.1-py3-none-any.whl"
Detecting file type...
File type is "Standard Python"
Extracting standard python attributes for upload
Error:  Trouble reading metadata from "dist/test_package-0.0.1-py3-none-any.whl". Is this a valid standard python package?
Error:  Trouble reading metadata from "dist/test_package-0.0.1-py3-none-any.whl". Is this a valid standard python package?
```

so I'll need Anaconda to clarify what the minimum metadata they need is. I'll keep this in draft until I get that verified.